src/functions/tag.js

```diff
@@ -4,21 +4,37 @@ import mathMLTree from "../mathMLTree";
 
 import * as mml from "../buildMathML";
 
+const pad = () => {
+    const padNode = new mathMLTree.MathNode("mtd", []);
+    padNode.setAttribute("width", "50%");
+    return padNode;
+};
+
 defineFunctionBuilders({
     type: "tag",
     mathmlBuilder(group, options) {
         const table = new mathMLTree.MathNode("mtable", [
-            new mathMLTree.MathNode("mlabeledtr", [
+            new mathMLTree.MathNode("mtr", [
```

Comment:
I'm late to the party here, but I wonder if (in a future PR) `mtr` should be `mlabeledtr` for `\tag`s? See https://www.w3.org/TR/MathML/chapter3.html#id.3.5.3.3

Reply:
@edemaine I agree, but only when and if the major browsers actually support `<mlabeledtr>`. At present, Firefox has baked-in browser CSS that hides the text inside a `<mlabeledtr>`. The Chromium MathML project's [project scope](https://mathml.igalia.com/project/) explicitly omits `<mlabeledtr>`.

Reply:
Oops, sorry, missed that in your original post. Thanks for investigating this!